lib/app/core/managers/audio_record_services.dart

```diff
@@ -5,15 +5,16 @@ import 'package:flutter_modular/flutter_modular.dart';
 import 'package:flutter_sound/flutter_sound.dart';
```

Comment:
Como ainda não temos a solução definitiva para os botões, e também que possivelmente essa classe precisa ser refatorada para separar as responsabilitades, poderia ignorar os erros de lint, o que acha?
```suggestion
// ignore_for_file: deprecated_member_use
import 'package:flutter_sound/flutter_sound.dart';
```

Reply:
Acredito que isto é uma má prática. Desativar uma regra definida em Linter. Isto vai retornar em toda a discussão de não elevar o 'info' a 'error' porque não tem solução para o 'info'.